### PR TITLE
Disabling check for conditional references

### PR DESCRIPTION
**Pull request by SergeyGaluzo**
$Import - Disabling check for conditional references in incremental mode

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Import/ImportResourceParser.cs [hunk 1]
```diff
@@ -67,7 +72,7 @@ private static void CheckConditionalReferenceInResource(Resource resource)
 
                 if (reference.Reference.Contains('?', StringComparison.Ordinal))
                 {
-                    throw new NotSupportedException("Conditional reference is not supported for $import.");
+                    throw new NotSupportedException("Conditional reference is not supported for initial $import.");
```
Comment (mikaelweave) [inline src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Import/ImportResourceParser.cs:75]: My only suggestion would be to use the enum name vs "initial" in this string. If more import modes are added later, this won't have to be updated.

**Review by mikaelweave — CHANGES_REQUESTED** [review]
One small comment - rest of the PR looks good. Will approve after the comment is addressed.

File: src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Import/ImportResourceParser.cs (hunk 1 above)
Comment (SergeyGaluzo) [reply src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Import/ImportResourceParser.cs]: Fixed.

**Review by mikaelweave — APPROVED** [review]
Looks good!

**Pull request merged**